packages/app/src/app/components/Preview/DevTools/Tests/index.tsx

```diff
@@ -201,7 +207,19 @@ class Tests extends React.Component<DevToolProps, State> {
 
   handleMessage = (data: SandboxMessage) => {
     if (data.type === 'done' && this.state.watching && !this.props.hidden) {
-      this.runAllTests();
+      let delay = 1000;
```

Comment:
This is good, I agree with the change. It would be good to add a comment explaining why you did this for the next readers 😄.

Reply:
done